internal/operators/odf/validations.go

```diff
@@ -164,14 +164,14 @@ func (o *operator) setStatusInsufficientResources(odfClusterResources *odfCluste
 }
 
 // count all disks of drive type ssd or hdd
-func (o *operator) getValidDiskCount(disks []*models.Disk, installationDiskID string) (int64, error) {
+func (o *operator) getValidDiskCount(disks []*models.Disk, installationDiskID string, mode odfDeploymentMode) (int64, error) {
 	var countDisks int64
 	var err error
 
 	for _, disk := range disks {
 		if (disk.DriveType == models.DriveTypeSSD || disk.DriveType == models.DriveTypeHDD) && installationDiskID != disk.ID && disk.SizeBytes != 0 {
 			if disk.SizeBytes < conversions.GbToBytes(o.config.ODFMinDiskSizeGB) {
-				err = fmt.Errorf("ODF requires all the non-bootable disks to be more than %d GB", o.config.ODFMinDiskSizeGB)
+				err = fmt.Errorf("Insufficient Resources to deploy ODF in non-bootable %s Mode. ODF requires a minimum of 3 hosts with 1 non-bootable disk on each host of size at least %d GB", mode, o.config.ODFMinDiskSizeGB)
```

Comment:
`Resources`, `%s` (`mode`) and `Mode` should not be capitalized

Reply:
Maybe `mode` should be capitalize as it goes together with `Compact` or `Standard` which are capitalized? @omertuc

Reply:
As I mentioned, both `Mode` and `%s` should not be capitalized